src/typeDefs/types.ts

```diff
@@ -10,6 +10,10 @@ export const types = gql`
     count: Int!
   }
 
+  type AggregateTag {
```

Comment:
This type is useless tbh. We can directly expose the tagsCount on User and Organization types, or directly inside their nested tags connection.

Reply:
Got it. Will remove the same.